app/models/user.js

```diff
@@ -72,7 +72,7 @@ module.exports = (sequelize, DataTypes) => {
       throw errors.savingError(err.errors);
     });
 
-  User.getAll = () => User.findAll().then(array => array);
+  User.getAll = () => User.findAll();
```

Comment:
This comment for future functionalities, always catch in this types of functions for errors related with db